source/assets/js/helpers/events.js

```diff
@@ -22,15 +23,18 @@ class WindowEventListener {
 
 		let events = {
 			resize: {
-				interval: 50
+				interval: 50,
+				throttleInterval: 10
```

Comment:
Setting the throttleInterval to 10ms will execute the callback sometimes more than once per frame which is not as performant as it could be. Consider using 16ms which results in roughly 60 FPS or try using [raf-throttle](https://www.npmjs.com/package/raf-throttle) for better performance since it uses the requestAnimationFrame as a throttle basis and no fixed timeout.

Reply:
As we talked: feel free to implement! :)